doc/source/reference/ufuncs.rst

```diff
@@ -299,7 +299,20 @@ advanced usage and will not typically be used.
 
     .. versionadded:: 1.6
 
-    The first output can provided as either a positional or a keyword parameter.
+    The first output can be provided as either a positional or a keyword
+    parameter. Keyword 'out' arguments are incompatible with positional
+    ones.
+
+    ..versionadded:: 1.10
+
+    The 'out' keyword argument is now expected to be a tuple with one entry
```

Comment:
I'd delete "now" here. Documentation should aim at those without previous knowledge; the `versionadded` will be enough for those who did know.